man/deepsea.1

```diff
@@ -0,0 +1,78 @@
+.TH deepsea 1 "DeepSea CLI man page"
+.SH NAME
+DeepSea CLI
+
+.SH SYNOPSIS
+deepsea [--version] [--help] [-l <log_level> | --log-level=<log_level>]
+        [--log-file=<path>]
+        <command> [<args>]
+
+.SH DESCRIPTION
+DeepSea CLI is a command-line-interface tool to run and visualize DeepSea's
+stages execution progress.
+
+Two modes are supported for visualizing a stage's execution progress.
+.RS
+.RE
+.B 1) monitoring mode:
+which allows to visualize the stage execution progress triggered by a salt-run
+command issued in another terminal session; and
```

Comment:
s/which allows to visualize/visualizes/

Reply:
visualizes the execution progress of a DeepSea stage triggered by a salt-run . . .